### PR TITLE
4455 Calendar blocking lambda fails if cal doesn't exist in Acuity

### PR DESCRIPTION
**Pull request by sophiemachin**

### CODE REVIEW

File: tests/unit_tests/test_main.py [hunk 1]
```diff
@@ -77,3 +77,7 @@ def test_get_target_calendar_ids(self):
         self.assertIn(
             (str(self.test_calendar["id"]), self.test_calendar["name"]), result
         )
+
+    def test_create_blocks_calendar_does_not_exist_in_dynamo_db(self):
```
Comment (sophiemachin) [inline tests/unit_tests/test_main.py:81]: Please check that this test fails if you remove the bug fix

Comment (afs25) [reply tests/unit_tests/test_main.py:81]: It doesn't I am afraid

File: tests/unit_tests/test_main.py [hunk 2]
```diff
@@ -77,3 +77,7 @@ def test_get_target_calendar_ids(self):
...
+        """ """
+        self.calendar_blocker.create_blocks()
```
Comment (afs25) [inline tests/unit_tests/test_main.py:83]: Delete this test

**Review by afs25 — APPROVED** [review]
Happy for you to merge after addressing comments

**Review by afs25 — APPROVED** [review]

**Pull request merged**